src/Navigation/styled.js

```diff
@@ -0,0 +1,35 @@
+import styled from "styled-components"
+
+export const Wrapper = styled.div`
+    max-width: 1920px;
+    background-color: ${({ theme }) => theme.color.almostBlack};
+    min-height: 93px;
```

Comment:
from Figma I understand that we have fixed header height values, especially on small devices ours is much higher